### PR TITLE
adding a global community disclaimer

### PR DESCRIPTION
**Pull request by tobiasd**
see issue #3894

### CODE REVIEW

File: doc/htconfig.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ Example: To set the directory value please add this line to your .htconfig.php:
 * **remove_multiplicated_lines** (Boolean) - If enabled, multiple linefeeds in items are stripped to a single one.
 * **show_unsupported_addons** (Boolean) - Show all addons including the unsupported ones.
 * **show_unsupported_themes** (Boolean) - Show all themes including the unsupported ones.
+* **show_global_community_hint** (Bbolean) - When the global community page is enabled, use this option to display a hint above the stream, that this is a collection of all public top-level postings that arrive on your node.
```
Comment (annando) [inline doc/htconfig.md]: ```Boolean``` not ```Bbolean```

File: mod/community.php [hunk 2]
```diff
@@ -83,9 +81,15 @@ function community_content(App $a, $update = 0) {
 
 	$o .= conversation($a, $s, 'community', $update);
 
-        $o .= alt_pager($a, count($r));
+	$o .= alt_pager($a, count($r));
 
-	return $o;
+	$t = get_markup_template( "community.tpl" );
```
Comment (annando) [inline mod/community.php]: Please don't use spaces here.

File: mod/community.php [hunk 3]
```diff
@@ -83,9 +81,15 @@ function community_content(App $a, $update = 0) {
...
+	return replace_macros($t, array(
+		'$content' => $o,
+		'$header' => t("Community"),
+		'$show_global_community_hint' => (Config::get('system', 'community_page_style') == CP_GLOBAL_COMMUNITY && Config::get('system', 'show_global_community_hint')),
+		'$global_community_hint' => t("This comunity stream shows all public messages arriving on this node. They do not reflect the opinion of the nodes users.")
```
Comment (annando) [inline mod/community.php]: This sounds strange. I guess that @MrPetovan or @zeroadam can help to improve it.

Comment (MrPetovan) [reply mod/community.php]: “This community stream shows all public posts received by this node. They may not reflect the opinions of this node’s users.”

File: mod/community.php [hunk 4]
```diff
@@ -83,12 +83,12 @@ function community_content(App $a, $update = 0) {
...
 	$o .= alt_pager($a, count($r));
 
-	$t = get_markup_template( "community.tpl" );
+	$t = get_markup_template("community.tpl");
 	return replace_macros($t, array(
 		'$content' => $o,
 		'$header' => t("Community"),
 		'$show_global_community_hint' => (Config::get('system', 'community_page_style') == CP_GLOBAL_COMMUNITY && Config::get('system', 'show_global_community_hint')),
-		'$global_community_hint' => t("This comunity stream shows all public messages arriving on this node. They do not reflect the opinion of the nodes users.")
+		'$global_community_hint' => t("“This community stream shows all public posts received by this node. They may not reflect the opinions of this node’s users.”")
```
Comment (MrPetovan) [inline mod/community.php]: Sorry for adding quotes!

Comment (annando) [inline mod/community.php]: Double ```""``` aren't intended, aren't they?

Comment (MrPetovan) [reply mod/community.php]: It’s a copy-paste issue from my reply 😅

**Comment by tobiasd** [comment]
On Fri, 10 Nov 2017 20:05:13 +0000 (UTC)
Michael Vogel <notifications@github.com> wrote:

> Double ```""``` aren't intended, aren't they?

*lol* no they are not ;-)

**Pull request merged**